utils/collect-updates/collect-updates.js

```diff
@@ -51,8 +49,10 @@ function collectUpdates(filteredPackages, packageGraph, execOpts, commandOptions
   let candidates;
 
   if (!committish || forced.has("*")) {
+    log.info("", `Assuming all packages changed`);
```

Comment:
I ported this file's changes in 9b8dd9c, please rebase when you can.

Reply:
I just merged the current master and resolved the hereby resulting merge conflict. Did you have anything else in mind? For rebasing I would have to either create a new branch or force push. Anyways, if you squash and merge this shouldn't make a difference.